### ISSUE TITLE
Support pure IPv6 environments (no IPv4)

### ISSUE THREAD

**Issue by tiraboschi** [lago-project/lago#770]
Currently IPv6 addresses for hosts are supported only appending a fixed prefix to the IPv4 addresses assigned by libvirt DHCP.
See: https://github.com/lago-project/lago/blob/master/lago/providers/libvirt/network.py#L223

Please support also pure IPv6 environments without IPv4 addresses.

**Comment by tiraboschi** [comment]
@dominikholler @sandrobonazzola

**Comment by mykaul** [comment]
That's not entirely true - just configure the VMs not to ask for IPv4 addresses. 
Of course, this is slightly more complex, as Lago relies heavily on SSH connectivity to the hosts.
But for some networks, we could do it. Perhaps it's a good start?

**Comment by tiraboschi** [comment]
I'm not sure it will be enough.
We have an explicit request to test, and keep testing, HE deployment on a pure IPv6 scenario with no IPv4 at all. We can cheat a bit here and there but I'm not sure we are going to test the same scenario.

**Comment by dankenigsberg** [comment]
Indeed, it would be best to use 3 VMs with only IPv6 addresses, just like the customer wants. However, I think it would be a good enough first step to "cheat" a bit, by preparing both ipv4 and ipv6 on the VMs, but using only the IPv6 ones in our code. It may miss a bug or two, but would make sure that all our own fields and arguments support ipv6. It also make it possible to develop the suite before (or in parallel) we fix this Lago issue.
